welcome/src/main/kotlin/edu/artic/welcome/WelcomeFragment.kt

```diff
@@ -35,5 +50,61 @@ class WelcomeFragment : BaseViewModelFragment<WelcomeViewModel>() {
             }
         }
 
+        val list = getTours()
+
+        context?.let {
+            tourSummaryRecyclerView.layoutManager = LinearLayoutManager(activity, LinearLayoutManager.HORIZONTAL, false)
+            tourSummaryRecyclerView.adapter = ToursAdapter(list, it)
+            welcomePreferencesManager = WelcomePreferencesManager(it)
+        }
+
+        /**
+         * Peak Animation.
+         * Scroll RecyclerView to the last item and back again to first item.
+         */
+        val peekedTourSummary = welcomePreferencesManager?.peekedTourSummary ?: false
```

Comment:
this should be handled by a viewmodel that injects the `WelcomePreferencesManager` and exposes methods for it. so we can easier test.

Reply:
Done.